docs/modules/kubernetes/pages/kubernetes-persistence.adoc

```diff
@@ -0,0 +1,41 @@
+= Running Hazelcast Enterprise with Persistence under Kubernetes
+:description: Hazelcast Enterprise members configured with persistence enabled can monitor the Kubernetes (K8s) context and automate Hazelcast cluster state management to ensure the optimal cluster behavior during shutdown and restart.
+:page-enterprise: true
+
+Hazelcast Enterprise members configured with xref:storage:configuring-persistence.adoc[persistence] enabled can monitor the Kubernetes (K8s) context and automate Hazelcast cluster state management to ensure the optimal cluster behavior during shutdown and restart. Specifically:
+
+ - During a cluster-wide shutdown, the Hazelcast cluster automatically switches the cluster to a `PASSIVE` xref:maintain-cluster:cluster-member-states.adoc#cluster-states[cluster state]. This offers the following advantages over the behavior in previous Hazelcast Platform releases (where Hazelcast always ran clusters in an `ACTIVE` state):
+   * No data migrations are performed, speeding up the cluster shutdown.
+   * No risk of out-of-memory exception due to migrations. When a cluster is in an `ACTIVE` state and Kubernetes applies the ordered shutdown of members, all data is eventually migrated to a single member (the last one in the shutdown sequence). This relieves you of the need to plan capacity for a single member to hold all the cluster data, or risk an out-of-memory exception, which was necessary with previous Hazelcast Platform releases.
+   * Persisted cluster metadata remain consistent across all members during shutdown. This consistency allows recovery from disk to proceed without unexpected metadata validation errors. These errors can result in the need for a xref:storage:triggering-force-start.adoc[force-start], wiping out persistent data from one or more members.
+ - During temporary loss of members (for example, with a rolling restart of the cluster or a pod being rescheduled by Kubernetes), Hazelcast cluster switches to a configurable cluster state (`FROZEN` or `NO_MIGRATION`) to ensure speedy recovery when the member rejoins the cluster.
+ - When scaling up or down, Hazelcast automatically switches to `ACTIVE` cluster state, so partitions are rebalanced and data is spread across all members.
+
+TIP: For a step-by-step tutorial that details how to deploy Hazelcast with persistence enabled, see https://docs.hazelcast.com/tutorials/hazelcast-platform-operator-external-backup-restore[Restore a Cluster from Cloud Storage with Hazelcast Platform Operator]
```

Comment:
I know you will add more details about the operator with upcoming PRs but still we can promote more Operator here. We can directly mention Operator beginning of page with Operator doc cross reference. WDYT?
https://docs.hazelcast.com/operator/latest/backup-restore

Reply:
Yeah, good suggestion - I'll add a reference at the top of this page.